packages/application-shell/src/components/application-shell/application-shell.js

```diff
@@ -335,6 +335,16 @@ RestrictedApplication.propTypes = {
   DEV_ONLY__loadNavbarMenuConfig: PropTypes.func,
 };
 
+const shallowlyCoerceBooleanValues = obj =>
```

Comment:
Nitpick: memoize based on `obj` otherwise the provider will re-render quite a bit.